HelpSource/Classes/UnitTest.schelp

```diff
@@ -26,6 +26,19 @@ code::
 UnitTest.gui
 ::
 
+METHOD:: run
+Run all methods whose names begin with code::test_::.
+
+code::
+TestUnitTest.new.run;
+::
+
+ARGUMENT:: reset
+If code::true::, first runs link::#*reset:: on the class.
```

Comment:
Maybe add something along these lines:
"This clears the data reported by method::report::"

Reply:
I'd prefer not to duplicate the documentation for `*reset` in two places; if in the future its behaviour changes then it increases the risk of this documentation getting left behind.  It's only a minor inconvenience to have to follow the link to find out the details.  But if you feel strongly in the other direction I'll change it.

Reply:
My point is that saying reset resets when true is not enough. We should be more specific about what this method does. For example, I can imagine someone thinking that they need to reset a test before running it a second time. If understand it correctly, this is not the case.

Reply:
Oh I see, yeah that makes sense.  So basically explain why/when anyone would want to use this.  Which raises the good question why/when anyone would want to use this, ideas welcome ;-)